tests/prague/eip7702_set_code_tx/test_set_code_txs_2.py

```diff
@@ -24,10 +24,10 @@
     TransactionException,
     compute_create_address,
 )
-from ethereum_test_tools.code.generators import CodeGasMeasure
-from ethereum_test_tools.vm.opcode import Opcodes as Op
+from ethereum_test_tools.tools_code.generators import CodeGasMeasure
```

Comment:
This should be updated to address Mario's comment:
https://github.com/ethereum/execution-spec-tests/pull/2133#discussion_r2341890678